app/controllers/slugs_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class SlugsController < ApplicationController
+  requires_login
+
+  MAX_SLUG_GENERATIONS_PER_MINUTE = 20
+
+  def generate
```

Comment:
I think we could have gone with `create` here and a POST route. I understand it's not exactly creating an object on the server, but in a way we are even though it's not persisted and it's just generally more expected (at least from my point of view).